apps/box/src/app/App.tsx

```diff
@@ -8,16 +8,23 @@ import { NavContainer } from '../navigation/NavContainer';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
 import { Platform, StatusBar, StyleSheet, UIManager } from 'react-native';
 import { BottomSheetModalProvider } from '@gorhom/bottom-sheet';
+import { useSettingsStore } from '../stores';
 
 if (Platform.OS === 'android') {
   if (UIManager.setLayoutAnimationEnabledExperimental) {
     UIManager.setLayoutAnimationEnabledExperimental(true);
   }
 }
 
+const fullTheme = {
+  light: fxLightTheme,
+  dark: fxDarkTheme,
+};
+
 export const App = () => {
   const [isDarkTheme] = React.useState<boolean>(true);
```

Comment:
remove this state variable and update the status bar logic below.

Reply:
Oh, weird it was removed but I think when I rebased with main it was added back in.

Reply:
Have fixed